py-data.csv

```diff
@@ -122,7 +122,7 @@ https://github.com/cbornet/python-httpproblem,0b08139277e9cba02339eeb0996c0910d0
 https://github.com/cbornet/python-httpproblem,0b08139277e9cba02339eeb0996c0910d047274a,tests/test_problem.py::test_problem_http_response[None-None-None-None-None-headers2-kwargs2-expected_body2-expected_headers2],OD-Vic,,,
 https://github.com/cbornet/python-httpproblem,0b08139277e9cba02339eeb0996c0910d047274a,tests/test_problem.py::test_problem_http_response[None-None-None-None-None-headers3-kwargs3-expected_body3-expected_headers3],OD-Vic,,,
 https://github.com/cbornet/python-httpproblem,0b08139277e9cba02339eeb0996c0910d047274a,tests/test_problem.py::test_problem_http_response[None-None-None-None-None-None-kwargs1-expected_body1-expected_headers1],OD-Vic,,,
-https://github.com/cbsinteractive/elemental.git,ca570c9f5fc43e20a4db23fc0137380f87be63f0,elemental/client_test.py::test_send_request_should_call_request_as_expected,OD-Vic,,,
+https://github.com/cbsinteractive/elemental.git,ca570c9f5fc43e20a4db23fc0137380f87be63f0,elemental/client_test.py::test_send_request_should_call_request_as_expected,OD-Vic,RepoArchived,,
```

Comment:
Please correct the repo name to not have `.git`. Can you check globally if some other rows have such `.git`? Moreover, could you change the checker to automatically check for such cases?

Reply:
Sorry, what checker are you referring to?

Reply:
Yes, change all 6 and squash your commits.